main/src/ca/uwaterloo/flix/language/ast/MonoAst.scala

```diff
@@ -20,15 +20,18 @@ import ca.uwaterloo.flix.language.ast.Ast.{Denotation, EliminatedBy, Source}
 import ca.uwaterloo.flix.language.ast.shared.Fixity
 import ca.uwaterloo.flix.language.phase.Monomorpher
 
+import java.lang.reflect.Field
+
 object MonoAst {
 
-  val empty: Root = Root(Map.empty, Map.empty, None, Set.empty, Map.empty)
+  val empty: Root = Root(Map.empty, Map.empty, None, Set.empty, Map.empty, Map.empty)
 
   case class Root(defs: Map[Symbol.DefnSym, Def],
                   effects: Map[Symbol.EffectSym, Effect],
                   entryPoint: Option[Symbol.DefnSym],
                   reachable: Set[Symbol.DefnSym],
-                  sources: Map[Source, SourceLocation])
+                  sources: Map[Source, SourceLocation],
+                  structs: Map[Symbol.StructSym, Struct])
```

Comment:
In every AST after typed, I think structs should come just after defs-- for consistency.

Reply:
Got it. We don't want to preserve the alphabetical ordering then?

Reply:
No. That was accidental here.

Reply:
It goes defs, enums, structs, and other stuff